docs/dvm-system-contract.md

```diff
@@ -28,4 +27,83 @@ The system contracts are deployed in various networks as follows.
 | Sha256        | 0x0000000000000000000000000000000000000002 |
 | Ripemd160     | 0x0000000000000000000000000000000000000003 |
 | Identity      | 0x0000000000000000000000000000000000000004 |
-| WithDraw      | 0x0000000000000000000000000000000000000015 |
\ No newline at end of file
+| WithDraw      | 0x0000000000000000000000000000000000000015 |
+
+> The Transfer contract is a superset of the Withdraw contract, and the Crab network's Withdraw contract will be upgraded to the Transfer contract in the future.
+
+## Contract Docs
+
+In order to be as compatible as possible with Ethereum and reduce the burden for application migration, the system contract currently consists of two types, the pre-compiled contracts compatible with Ethereum(ECRecover, Sha256, Ripemd160, Identity) and the Pangolin-specific pre-compiled contract Transfer.
+
+### Transfer Contract
+
+Transfer is a Pangolin-specific pre-compiled contract with address `0x0000000000000000000000000000000000000015` (reserved for other Ethereum contracts) that is used to transfer value between DVM and Substrate account. The current Transfer contract has two main features:
+
+1. support DVM PRING asset transfer to Substrate account. 
+2. support KTON asset transfer between DVM and Substrate account.
```

Comment:
1. Transfer PRING from DVM to Substrate account.
2. Transfer PKTON between DVM and Substrate account.